Description: Add a proper schema version to network_data.json
 What we have there is not a schema URI. Having a proper URI allows
 auto-detection of the schema version (draft-07 in this case).
Author: Dmitry Tantsur <dtantsur@protonmail.com>
Date: Thu, 14 Jul 2022 17:59:21 +0200
Change-Id: I208a05c7ba64a21ed8cde7bc1ebae2b5878109e0
Origin: upstream, https://review.opendev.org/c/openstack/nova/+/849881
Last-Update: 2022-07-14

diff --git a/doc/api_schemas/network_data.json b/doc/api_schemas/network_data.json
index f980973..c05f068 100644
--- a/doc/api_schemas/network_data.json
+++ b/doc/api_schemas/network_data.json
@@ -1,5 +1,5 @@
 {
-  "$schema": "http://openstack.org/nova/network_data.json#",
+  "$schema": "http://json-schema.org/draft-07/schema#",
   "id": "http://openstack.org/nova/network_data.json",
   "type": "object",
   "title": "OpenStack Nova network metadata schema",
